Python/bytecodes.c

```diff
@@ -1218,19 +1218,33 @@ dummy_func(
             tstate->current_frame = frame->previous;
             assert(!_PyErr_Occurred(tstate));
             PyObject *result = PyStackRef_AsPyObjectSteal(retval);
+            if (IS_JIT_TRACING()) {
+#if _Py_TIER2
+                _PyJit_translate_single_bytecode_to_trace(tstate, frame, next_instr);
```

Comment:
```suggestion
                _PyJit_translate_single_bytecode_to_trace(tstate, frame, NULL);
```
`next_instr` is invalid here.

Reply:
Don't you need to add a `_DEOPT` to the trace?

Reply:
it automatically adds an _EXIT_TRACE, though I think you're right it should be a deopt.